site/content/blog/2021-10-01-whats-new-in-svelte-october-2021.md

```diff
@@ -0,0 +1,99 @@
+---
+title: What's new in Svelte: October 2021
+description: A whole year of "What's new in Svelte"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+Hey y'all 👋 It's been 1 year since "What's new in Svelte" started being cross-posted to the Svelte blog. I wanted to take this moment at the top to thank all of you for reading and for all the contributors every month. From the maintainers to everyone who posts their work in the Discord and Reddit, it's amazing to witness all the effort that goes into making the Svelte community great.
+
+Keep up the good work, everyone! Now, let's dive into this month's news...
+
+## New around Svelte
+
+- New additions to Svelte's export map now expose no-op versions of lifecycle functions for SSR (Svelte **3.43.0**)
+- Custom components with a `src` attribute no longer break `svelte-native` builds (Svelte **3.42.4**)
+- Svelte plugin users without [the TypeScript plugin](https://www.npmjs.com/package/typescript-svelte-plugin) enabled will now be prompted to enable it. It enhances TypeScript and JavaScript files with additional intellisense to interact with Svelte files. [Please leave feedback](https://github.com/sveltejs/language-tools/issues/580) if you are using it (Svelte extensions **105.4.0**)
+- Event modifiers have been added to intellisense as autocomplete and hover info (Svelte extensions **105.4.0**)
+- TypeScript users no longer have to strictly separate type imports and value imports when using Svelte version 3.39 or higher and `svelte-preprocess` version 4.9.5 or higher. This means you can now write `import { MyInterface, myValue } from './somewhere'` instead of `import type { MyInterface } from './somewhere'; import { myValue } from './somewhere'`. Huge thanks to community member [@SomaticIT](https://github.com/SomaticIT) who mainly implemented this!
+
+For a full list of features and bug fixes, check out the [Svelte changelog](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md).
+
+## SvelteKit Updates
+
+Nearly 100 PRs committed again this past month, but there's still lots to do and Svelte maintainers are [looking for help getting SvelteKit to 1.0](https://github.com/sveltejs/kit/issues/2100). Antony said it well in a [recent comment](https://github.com/sveltejs/kit/issues/2100#issuecomment-895446285) on the issue: 
+
+> If you think you are too n00b to contribute (you're not), then add tests, or write tests for the feature you want to add, before you add it! Start small, and learn the codebase that way.
+
+If you'd like to help, please consider working on any of the [1.0 milestone issues labeled with "help wanted"](https://github.com/sveltejs/kit/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0+label%3A%22help+wanted%22).
+
+Notable SvelteKit improvements this month include...
+
+- Service workers are now allowed to access files using the \$lib alias ([#2326](https://github.com/sveltejs/kit/pull/2326))
```

Comment:
Is the backslash here have some effect? I'd expect $lib to be in backticks with no backslash

Reply:
I just copied it from the git log. Maybe it was an escape character 🤔